src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
I like how your use the `is` prefix here to make your boolean attribute sound like valid booleans.